src/services/parseFilesAsAttachments.ts

```diff
@@ -1,5 +1,36 @@
 import { Sentry } from '@oneblink/apps'
 import loadImage from 'blueimp-load-image'
+import { AttachmentNew } from '../types/attachments'
+
+async function correctFileOrientationAsCanvas(
+  file: File | Blob,
+): Promise<HTMLCanvasElement> {
+  // @ts-expect-error For some reason, the types do not include this function returning a promise
+  const imageMetaData: loadImage.MetaData = await loadImage.parseMetaData(file)
+
+  console.log(
+    'Loading image onto canvas to correct orientation using image meta data',
+    imageMetaData,
+  )
+  const orientation = imageMetaData.exif?.get('Orientation')
```

Comment:
I just learnt that if `orientation` is `undefined` or `1`, we can skip this step. Which should be true most of the time. Should be much faster if we don't have to load the image onto a canvas.